docs/reference/api/linea-estimategas.mdx

```diff
@@ -0,0 +1,71 @@
+---
+description: Reference content for the linea_estimateGas method.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# `linea_estimateGas`
+
+Generates and returns an estimate of how much gas is necessary to allow the transaction to complete and be published on Ethereum.
+The transaction will not be added to the blockchain.
+
+For more information about estimating gas, and how this API formulates the transaction costs, see the
+[Estimate transaction costs](../../build-on-linea/gas-fees.mdx) topic.
+
+:::note We recommend using linea_estimateGas for more accurate results
+
+The API uses the same inputs as the standard [eth_estimateGas](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_estimategas),
+but returns the recommended gas limit, the base fee per gas, and the priority fee per gas.
+
+:::
+
+## Parameters
+
+- `TRANSACTION CALL OBJECT` _\[required]_
+  - `from`: _\[optional]_ 20 Bytes - The address the transaction is sent from.
+  - `to`: _\[optional]_ 20 Bytes - The address the transaction is directed to.
+  - `gas`: _\[optional]_ Hexadecimal value of the gas provided for the transaction execution. `linea_estimateGas` consumes
+      zero gas, but this parameter may be needed by some executions.
+  - `gasPrice`: _\[optional]_ Hexadecimal value of the gas price used for each paid gas.
+  - `maxPriorityFeePerGas`: _\[optional]_ Maximum fee, in Wei, the sender is willing to pay per gas above the base fee.
```

Comment:
"Wei" is capitalized on this page, but not capitalized on the "Estimate transaction costs" page. We should choose one style.

Reply:
Should be "wei", like cent or dollar.